preprocessors/openstreetmap/osm_service.py

```diff
@@ -933,6 +934,10 @@ def validate(schema, data, resolver, json_message, error_code):
         return jsonify(json_message), error_code
     return None
 
+# We don't currently support embedded OSM Map, so a request from
```

Comment:
Don't know if this comment is necessary/appropriate. If we are able to get lat/lon from OSM maps on the client, it will be transparent to the preprocessor.

Recommend: just note what input types are supported, ideally at the top of the file and not several hundred lines in.